### PR TITLE
Greek mask effort

### PR DESCRIPTION
**Pull request by MorganGR**
I'm hoping we can help these guys also as their funding is limited and they are producing hundreds of masks.
Potentially useful for the whole region of Northern Greece/Balkan states. 

Email: contact@covid19response.gr
Coordinator: Dimitris Moustakas

### CODE REVIEW

**Comment by bneiluj** [comment]
Hi @MorganGR, we need more information on the mask: image, links, media etc

File: Greek mask effort [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Team based in Thessaloniki, Greece is producing masks for medical use.
+Translated page here: https://translate.google.com/translate?hl=&sl=el&tl=en&u=https%3A%2F%2Fcovid19response.gr%2Fproject1.html
```
Comment (bneiluj) [inline Greek mask effort:2]: I get: `Server Error 404 Page Not Found`on the link

Comment (bneiluj) [reply Greek mask effort:2]: Also, is this solution any better than the Prusa Protective Face Shield - RC3, 3D printed face shields for medics and professionals by Prusa? 

It's easier for us to support the printing of the Prusa as it has been medically tested.

Comment (MorganGR) [reply Greek mask effort:2]: @bneiluj
bneiluj 9 minutes ago Owner
I get: Server Error 404 Page Not Foundon the link

@MorganGR	Reply…

Original Page here: https://covid19response.gr/project1.html

Comment (MorganGR) [reply Greek mask effort:2]: It is based on the Prusa version (and improved)

**Comment by MorganGR** [comment]
The Prusa design is too slow for the Greek mask effort. Last Friday 50 masks were delivered to Kozani hospital.
Translated Letter and Original Letter.

[scan20150204094139.pdf](https://github.com/bneiluj/covid-19-open-source-masks/files/4442566/scan20150204094139.pdf)
91844410_834475487059562_4459853031818657792_n
[ThankYouLetterGR.pdf](https://github.com/bneiluj/covid-19-open-source-masks/files/4442703/ThankYouLetterGR.pdf)

**Comment by bneiluj** [comment]
Is this mask part of the README ? https://github.com/bneiluj/covid-19-open-source-masks

**Comment by MorganGR** [comment]
Up to you I can do either! Whichever is required!

**Comment by bneiluj** [comment]
Grant can't be approved on masks that are not inside the REAMDE for security reasons. And the grant are for printing solutions. 

Why not using `3D printed visor, completed with a transparent sheet of paper` or `Laser cut visor, completed with a transparent sheet of paper` from the README ?

**Comment by MorganGR** [comment]
No problem thanks! 
This seems to have been a successful design as the Greek team have so far produced 20,000 masks all delivered. The 3D printing element slowed down their production so a clear "head band" method was used at the laser cutting facility.

**Comment by MorganGR** [comment]
Thanks for taking a look!

**Pull request closed without merging**